### PR TITLE
rustdoc: avoid including `<li>` tags in item table short desc

### PR DESCRIPTION
**Pull request by notriddle**
Fixes a bug seen at https://docs.rs/gl_constants/0.1.1/gl_constants/index.html

### CODE REVIEW

File: tests/rustdoc/item-desc-list-at-start.rs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// @has item_desc_list_at_start/index.html
+// @count - '//ul[@class="item-table"]/li/div/li' 0
```
Comment (GuillaumeGomez) [inline tests/rustdoc/item-desc-list-at-start.rs:2]: Wouldn't it be better to also include a snapshot check as well? They are quite useful to be absolutely sure that nothing changed in a given location.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks! r=me once CI pass

**Comment by Lokathor** [comment]
Once this merges can someone on the team also redo the docs of my `gl_constants` crate?

**Comment by GuillaumeGomez** [comment]
Please ask on the docs-rs chan on zulip.

**Pull request merged**